upup/pkg/fi/ca.go

```diff
@@ -108,6 +108,16 @@ type SSHCredentialStore interface {
 	FindSSHPublicKeys(name string) ([]*kops.SSHCredential, error)
 }
 
+// LoadPEMPrivateKey public
+func LoadPEMPrivateKey(pemData []byte) (*pki.PrivateKey, error) {
```

Comment:
I'm a little confused by the Load / Parse switch here - is there a distinction being made?

Reply:
that was actually not needed anymore since we had a function in pkg/pki/privatekey.go doing what i needed